src/storage-types/src/sources.proto

```diff
@@ -7,6 +7,10 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+// buf breaking: ignore (a compatible change: Field "3" on message
+// "ProtoSourceExportStatementDetails" changed type from
+// "google.protobuf.Empty" to "ProtoLoadGeneratorSourceExportStatementDetails".
```

Comment:
Do we gain anything by reusing the previous field id? I believe if the 3 were to be changed to 4 everything would compile without adding these buf directives

Reply:
Yes unfortunately we need the field-id to be the same for the migration to work, since we already have some loadgen subsource statements stored with the `loadgen` `kind`, and if we were to introduce a field `4` we'd still need to look for field `3` to know if this was a loadgen subsource that needs migrating.

I'll put out a follow up PR once this merges that removes the buf breaking annotation so it's not left here for more than 1 day